src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -1330,6 +1330,17 @@ class TestLibrary(SystemTests):
     def session_creation_kwargs(self):
         return {}
 
+    def test_enable_match_fail_combination(self, multi_instrument_session):
+        import nisync
+
+        test_files_folder = 'simple_pattern'
+        multi_instrument_session.load_pin_map(self.get_test_file_path(test_files_folder, "pin_map.pinmap"))
```

Comment:
If it doesn't work with this pattern, we ship an example for LabVIEW that we can borrow files from.

Reply:
I decided to just use the config files from the LabVIEW example.